App.tsx

```diff
@@ -180,25 +190,23 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                 <InfoListItem
                     title={'South Hills Farm'}
                     subtitle={'Offline'}
-                    backgroundColor={theme.dark ? PXBColors.darkBlack[100] : 'white'}
-                    statusColor={PXBColors.orange[500]}
+                    statusColor={PXBColors.red[500]}
                     divider={'full'}
                     IconClass={Pie}
                     iconAlign={'center'}
                     hidePadding={false}
-                    fontColor={PXBColors.red[500]}
+                    fontColor={theme.dark ? PXBColors.red[200] : PXBColors.red[500]}
                     rightComponent={<ChannelValue value={15} units={'A'} color={theme.colors.text} />}
                     onPress={(): void => {
                         /* do nothing */
                     }}
                 />
                 <InfoListItem
                     title={'Cherry East'}
-                    backgroundColor={theme.dark ? PXBColors.darkBlack[100] : 'white'}
                     subtitle={['DG 100', 'EDR 5000', 'Online']}
                     subtitleSeparator={'/'}
                     hidePadding
-                    rightComponent={<ListItemTag label={'ACTIVE'} />}
+                    rightComponent={<ListItemTag label={'ACTIVE'} theme={theme.dark ? blueDarkAlt : {}} />}
```

Comment:
Shall we write a wrapper for ListItemTag?

Reply:
I think this one is just preference since the end-user can make this any color they want. Is there a standard for what colors a listitemtag should be? If you want me to throw a wrapper in for this that's no problem though.